src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar_js.html.twig

```diff
@@ -13,87 +13,98 @@
+                        document.getElementById('sfToolbarClearer-' + token).style.display = 'block';
+                        document.getElementById('sfMiniToolbar-' + token).style.display = 'none';
+                    }
 
-                /* Handle toolbar-info position */
-                var toolbarBlocks = [].slice.call(el.querySelectorAll('.sf-toolbar-block'));
-                for (i = 0; i < toolbarBlocks.length; ++i) {
-                    toolbarBlocks[i].onmouseover = function () {
-                        var toolbarInfo = this.querySelectorAll('.sf-toolbar-info')[0];
-                        var pageWidth = document.body.clientWidth;
-                        var elementWidth = toolbarInfo.offsetWidth;
-                        var leftValue = (elementWidth + this.offsetLeft) - pageWidth;
-                        var rightValue = (elementWidth + (pageWidth - this.offsetLeft)) - pageWidth;
+                    /* Handle toolbar-info position */
+                    var toolbarBlocks = [].slice.call(el.querySelectorAll('.sf-toolbar-block'));
+                    for (i = 0; i < toolbarBlocks.length; ++i) {
+                        toolbarBlocks[i].onmouseover = function () {
+                            var toolbarInfo = this.querySelectorAll('.sf-toolbar-info')[0];
+                            var pageWidth = document.body.clientWidth;
+                            var elementWidth = toolbarInfo.offsetWidth;
+                            var leftValue = (elementWidth + this.offsetLeft) - pageWidth;
+                            var rightValue = (elementWidth + (pageWidth - this.offsetLeft)) - pageWidth;
 
-                        /* Reset right and left value, useful on window resize */
-                        toolbarInfo.style.right = '';
-                        toolbarInfo.style.left = '';
+                            /* Reset right and left value, useful on window resize */
+                            toolbarInfo.style.right = '';
+                            toolbarInfo.style.left = '';
 
-                        if (elementWidth > pageWidth) {
-                            toolbarInfo.style.left = 0;
-                        }
-                        else if (leftValue > 0 && rightValue > 0) {
-                            toolbarInfo.style.right = (rightValue * -1) + 'px';
-                        } else if (leftValue < 0) {
-                            toolbarInfo.style.left = 0;
+                            if (elementWidth > pageWidth) {
+                                toolbarInfo.style.left = 0;
+                            }
+                            else if (leftValue > 0 && rightValue > 0) {
+                                toolbarInfo.style.right = (rightValue * -1) + 'px';
+                            } else if (leftValue < 0) {
+                                toolbarInfo.style.left = 0;
+                            } else {
+                                toolbarInfo.style.right = '0px';
```

Comment:
Why ' 0px` here and just `0` on the previous line? Sorry if this is a noob question.